static/app/components/modals/widgetViewerModal.spec.tsx

```diff
@@ -51,6 +52,20 @@ async function renderModal({
   seriesResultsType?: Record<string, AggregationOutputType>;
   tableData?: TableDataWithTitle[];
 }) {
+  const dashboardLegendUtils = new DashboardLegendEncoderDecoder({
+    location: router.location,
+    dashboard: {
+      id: 'new',
+      title: 'Dashboard',
+      createdBy: undefined,
+      dateCreated: '2020-01-01T00:00:00.000Z',
+      widgets: [widget],
+      projects: [],
+      filters: {},
+    },
```

Comment:
I think `dashboard: DashboardFixture()` would be fine here. Applies to all the other spec files, too! If you can use the fixtures, it'd be a lot cleaner (I saw you did in one spot)